projects/storefrontlib/src/lib/product/components/product-details/product-summary/product-summary.component.html

```diff
@@ -1,24 +1,42 @@
-<div class="cx-product-summary">
-  <div class="cx-product-summary__price cx-product-summary__price" aria-label="new item price">
-    {{product?.price?.formattedValue}}
-  </div>
-
-  <div class="cx-product-summary__description">
-    <p [innerHTML]="product?.summary"> </p>
-  </div>
+<div class="product-summary">
```

Comment:
We don't need the div with product-summary class. We already have the component selector. The styles for `cx-product-summary` can be selected by `:host`.